changelogs/fragments/168_classic.yml

```
@@ -0,0 +1,6 @@
+---
+minor_changes:
```

Hmm, since we have that section, let's use it ;)
```suggestion
removed_features:
```